lib/auth/assist/assistv1/test/service_test.go

```diff
@@ -268,6 +275,44 @@ func TestService_InsertAssistantMessage(t *testing.T) {
 	}
 }
 
+func TestService_SearchUnifiedResources(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		username    string
+		req         *assistpb.SearchUnifiedResourcesRequest
+		wantErr     assert.ErrorAssertionFunc
+		returnedLen int
+	}{
+		{
+			username: defaultUser,
+			req: &assistpb.SearchUnifiedResourcesRequest{
+				Kinds: []string{types.KindNode},
+			},
+			wantErr:     assert.NoError,
+			returnedLen: 2,
+		},
+		{
+			username: noAccessUser,
+			req: &assistpb.SearchUnifiedResourcesRequest{
+				Kinds: []string{types.KindNode},
+			},
+			wantErr:     assert.NoError,
+			returnedLen: 0,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.username, func(t *testing.T) {
+			ctxs, svc := initSvc(t)
+			time.Sleep(time.Second * 5)
```

Comment:
Hard pass. This seems to work and makes this test to run in 100ms instead of 10s
```go
	for _, tt := range tests {
		tt := tt
		t.Run(tt.username, func(t *testing.T) {
			t.Parallel()

			ctxs, svc := initSvc(t)
			require.Eventually(t, func() bool {
				resp, err := svc.SearchUnifiedResources(ctxs[tt.username], tt.req)
				tt.wantErr(t, err)
				return tt.returnedLen == len(resp.GetResources())
			}, 5*time.Second, 100*time.Millisecond)
		})
	}
```

Reply:
Thanks! Fixed.